include/nuttx/wireless/ioctl.h

```diff
@@ -48,67 +48,214 @@
  * the open() interface.
  ****************************************************************************/
 
-#define WLIOC_SETRADIOFREQ  _WLCIOC(0x0001)  /* arg: Pointer to uint32_t, */
-                                             /* frequency value (in MHz) */
-#define WLIOC_GETRADIOFREQ  _WLCIOC(0x0002)  /* arg: Pointer to uint32_t, */
-                                             /* frequency value (in MHz) */
-#define WLIOC_SETADDR       _WLCIOC(0x0003)  /* arg: Pointer to address value, format
-                                              * of the address is driver specific */
-#define WLIOC_GETADDR       _WLCIOC(0x0004)  /* arg: Pointer to address value, format
-                                              * of the address is driver specific */
-#define WLIOC_SETTXPOWER    _WLCIOC(0x0005)  /* arg: Pointer to int32_t, */
-                                             /* output power (in 0.01 dBm) */
-#define WLIOC_GETTXPOWER    _WLCIOC(0x0006)  /* arg: Pointer to int32_t, */
-                                             /* output power (in 0.01 dBm) */
+/****************************************************************************
+ * RF common IOCTL commands
+ ****************************************************************************/
+
+/* Offsets */
+
+#define _WLIOC_COM_OFFS 1
```

Comment:
let's align _WLIOC_XXX_OFFS to the second column of line 62
```suggestion
#define _WLIOC_COM_OFF        S 1
```

Reply:
Good catch about the FAR. That still has to get into my muscle memory. Yeah sure about the alignments. Somewhere in my mind told me that having a different alignment for this section would keep a visual separation between the commands and these irrelevant to the user offsets. But for the sake of consistency ill just align them like everything else.